### ISSUE TITLE
beta.18 and beta.19 - Icons are disappearing

### ISSUE THREAD

**Issue by peterennis** [ionic-team/ionic#16775]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

**Expected behavior:**

**Steps to reproduce:**

![emptyicon](https://user-images.githubusercontent.com/140737/50062043-94b33380-0159-11e9-80d8-0898945db225.gif)

I also do not want to see the popup in the lower left that shows the address.

**Related code:**

Ref: https://github.com/peterennis/aetabs

```
insert short code snippets here
```

**Other information:**

**Ionic info:** 

```
insert the output from ionic info here
```

**Comment by peterennis** [comment]
```
C:\ae\adaept.com\aetabs>ionic info

Ionic:

   ionic (Ionic CLI)             : 4.5.0 (C:\Users\peter\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.0.0-beta.19
   @angular-devkit/build-angular : 0.11.3
   @angular-devkit/schematics    : 7.1.3
   @angular/cli                  : 7.1.3
   @ionic/angular-toolkit        : 1.2.0

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : none
   Cordova Plugins       : no whitelisted plugins (2 plugins total)

System:

   NodeJS : v10.14.1 (C:\Program Files\nodejs\node.exe)
   npm    : 6.4.1
   OS     : Windows 10
```

**Comment by paulstelzer** [comment]
Duplicate of #16761 and fixed by #16766
